src/FeatureFlagsProvider.php

```diff
@@ -145,4 +145,11 @@ private function registerCommands()
     {
         $this->commands(SyncFlags::class);
     }
+
+    private function provideInterfaces()
+    {
+        $this->app->bind(
+            'FriendsOfCat\LaravelFeatureFlags\FeatureFlagsUserRoles'
+        );
```

Comment:
I don't think this bind is doing anything?

Reply:
will remove!